api/envoy/extensions/upstreams/http/v3/http_protocol_options.proto

```diff
@@ -150,7 +150,11 @@ message HttpProtocolOptions {
     AutoHttpConfig auto_config = 5;
   }
 
-  // [#not-implemented-hide:]
+  // .. warning::
+  //   HTTP filters are not supported by default.
+  //   This warning will be removed as support moves beyond alpha.
+  //
```

Comment:
IMO this is kind of confusing. This is upstream only I guess? Or is this going to be downstream as well? If it's upstream only shouldn't this be in the upstream options above? Also per my comment below are we going to allow per-route upstream filter chains? I guess stepping back can we enumerate all of the different ways we eventually want to configure this and then we can sort out the best API?

Reply:
sorry yeah, this is where we configure the filter chain for upstream filters.  As commented below I had to unhide for now because of the presubmit checks.  

This is the moral equivalent of the filter list in the upstream HCM, only instead of these filters being applied to traffic on that HCM/listener, they're applied to streams in that cluter.